### PR TITLE
Forbidden no grouping mv on aggregation table

### LINKED ISSUE #4316: Create no grouping materialized view on agg table
**Describe the bug**
When user create no grouping materialized view on agg table, the job will be executed successfully and the mv keys type is incorrect.

**To Reproduce**
1. create table 
```
CREATE TABLE `test_sys_materialized_view_2_test_create_agg_mv_tb_agg` (

  `k0` boolean NOT NULL COMMENT "",

  `k1` tinyint(4) NOT NULL COMMENT "",

  `k2` smallint(6) NOT NULL COMMENT "",

  `k3` int(11) NOT NULL COMMENT "",

  `k4` bigint(20) NOT NULL COMMENT "",

  `k5` largeint(40) NOT NULL COMMENT "",

  `k6` decimal(9, 3) NOT NULL COMMENT "",

  `k7` char(5) NOT NULL COMMENT "",

  `k8` date NOT NULL COMMENT "",

  `k9` datetime NOT NULL COMMENT "",

  `k10` varchar(20) NOT NULL COMMENT "",

  `k11` double MAX NOT NULL COMMENT "",

  `k12` float SUM NOT NULL COMMENT "",

  `k13` hll HLL_UNION NOT NULL COMMENT "",

  `k14` bitmap BITMAP_UNION NOT NULL COMMENT ""

) ENGINE=OLAP

AGGREGATE KEY(`k0`, `k1`, `k2`, `k3`, `k4`, `k5`, `k6`, `k7`, `k8`, `k9`, `k10`)

COMMENT "OLAP"

DISTRIBUTED BY HASH(`k4`, `k5`) BUCKETS 10

PROPERTIES (

"replication_num" = "3",

"in_memory" = "false",

"storage_format" = "DEFAULT"

);
```

2.  create mv

```
CREATE MATERIALIZED VIEW mv1 AS select k6, k7 from test_sys_materialized_view_2_test_create_agg_mv_tb_agg
```

3. 

image

**Expected behavior**

throw exception when creating mv.

error_msg: `The materialized view of aggregation table must has grouping columns`

### PR DESCRIPTION
**Pull request by EmmyMiao87**
## Proposed changes
If user wants to create a no grouping mv on aggregation table, the doris will thrown exception.
The correct approach is that explicit declare the grouping column.
For example:
Agg table: k1, k2, sum(k3)
Create materialized view stmt: select k1, k2 from agg_table group by k1, k2.

Fixed #4316

Change-Id: I0851ffd5450ad0655741dfb20ead891e9aaf6015

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation Update (if none of the other choices apply)
- [] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have create an issue on #ISSUE, and have described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [] If this change need a document change, I have updated the document
- [] Any dependent changes have been merged

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java [hunk 1]
```diff
@@ -432,6 +432,9 @@ private RollupJobV2 createMaterializedViewJob(String mvName, String baseIndexNam
         List<Column> newMVColumns = Lists.newArrayList();
         int numOfKeys = 0;
         if (olapTable.getKeysType().isAggregationFamily()) {
+            if (addMVClause.getMVKeysType() != KeysType.AGG_KEYS) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java:435]: Do we allowed to create mv on unique key table?
if yes, how to create it?

**Review by morningman — COMMENTED** [review]
UT failed

File: fe/fe-core/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java (hunk 1 above)
Comment (EmmyMiao87) [reply fe/fe-core/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java]: We allowed to create mv on unique key if and only if materialized view of full key column.

**Review by morningman — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java (hunk 1 above)
Comment (liuzhizu) [reply fe/fe-core/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java:435]: the unique key table has unique feature , we need mv to aggregate  such as  bitmap_union(to_bitmap(user_id)),but  it's wrong 。
example:

CREATE TABLE t
(
    event_day DATE,
    momo_id INT ,
    event_name VARCHAR(100),
    access_timestamp BIGINT
)
UNIQUE KEY(event_day,momo_id, event_name,access_timestamp)
PARTITION BY RANGE(event_day) ()
DISTRIBUTED BY HASH(user_id)
PROPERTIES
(
    "dynamic_partition.enable" = "true",
    "dynamic_partition.time_unit" = "DAY",
    "dynamic_partition.end" = "3",
    "dynamic_partition.prefix" = "p",
    "dynamic_partition.buckets" = "8",
    "storage_type"="column"
);

create materialized view mv as
select  event_day,bitmap_union(to_bitmap(user_id))  from t group by event_day ;

**Pull request merged**